static/js/tools/timeline/chart.tsx

```diff
@@ -115,30 +118,30 @@ class Chart extends Component<ChartPropsType> {
     // provide a key for style look up.
     const placeName = Object.values(this.props.placeNames)[0];
     const deltaCheckboxId = `delta-cb-${this.props.mprop}`;
-    const perCapitaCheckboxId = `pc-cb-${this.props.mprop}`;
+    const ratioCheckboxId = `pc-cb-${this.props.mprop}`;
     return (
       <div className="card">
         <div ref={this.svgContainer} className="chart-svg"></div>
         <div className="chart-options">
           <span className="chart-option">
-            <label htmlFor={perCapitaCheckboxId}>Per capita</label>
             <button
-              id={perCapitaCheckboxId}
+              id={ratioCheckboxId}
               className={
-                this.props.perCapita
-                  ? "option-checkbox checked"
-                  : "option-checkbox"
+                this.props.pc ? "option-checkbox checked" : "option-checkbox"
               }
               onClick={() => {
-                setChartOption(this.props.mprop, "pc", !this.props.perCapita);
+                setChartOption(this.props.mprop, "pc", !this.props.pc);
               }}
             ></button>
-            <a href="/faq#perCapita">
-              <span> *</span>
-            </a>
+            <label htmlFor={ratioCheckboxId}>Ratio Of </label>
+            <input
+              ref={this.denomInput}
+              disabled={!this.props.pc}
+              placeholder={this.props.denom}
+              onChange={(evt) => this.handleDenomInput(evt)}
```

Comment:
I agree with using onBlur because as a user, if I enter something in the box and then click outside the box, I would expect it to "submit" that value.

Maybe also allow the enter key to "submit" the value to have it be consistent with map tool.

Reply:
That's better! I have never used it before..